modules/vaos/app/policies/vaos_policy.rb

```diff
@@ -2,7 +2,7 @@
 
 VAOSPolicy = Struct.new(:user, :vaos) do
   def access?
-    Flipper.enabled?('va_online_scheduling', user) && user.loa3?
```

Comment:
this seems right to remove but have we confirmed with VAOS that its won't cause any issues?

Reply:
Waiting for confirmation